process.sh

```diff
@@ -225,27 +225,44 @@ process_mermaid() {
   # to be done on the client-side.
 
   CSV_IN="$INPUT/data.csv"
-  PNG_IN="$INPUT/bg.png"
 
   CLI_ARGS="--csv_file $CSV_IN"
   add_arg 'cells' 'mermaid'
   add_arg 'molecules' 'mermaid'
+  add_arg 'images' 'mermaid'
```

Comment:
mermaid.png is  50KB... In the fixtures, let's replace it with a 10x10 image like we have for Linnarsson.

Reply:
Fixed.